server/constants/error.ts

```diff
@@ -1,5 +1,7 @@
 export enum ApiErrorCode {
   InvalidUrl = 'INVALID_URL',
   InvalidCredentials = 'INVALID_CREDENTIALS',
+  InvalidAuthToken = 'INVALID_AUTH_TOKEN',
   NotAdmin = 'NOT_ADMIN',
+  UNKNOWN = 'UNKNOWN',
```

Comment:
Not consistent with case?

Reply:
Oh yeah!
I was half asleep when I made this change 💀